modules/apps/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/review_uad_data.jsp

```diff
@@ -185,20 +185,22 @@ renderResponse.setTitle(StringBundler.concat(selectedUser.getFullName(), " - ",
 		<clay:col
 			lg="9"
 		>
-			<div class="sheet">
-				<div class="sheet-header">
+			<clay:sheet
+				size="<%= null %>"
```

Comment:
I cannot see this JSP:
http://localhost:8080/group/control_panel/manage?p_p_id=com_liferay_user_associated_data_web_portlet_UserAssociatedData

the compilation works fine, @jbalsas could you tell me if this is correct?